ci/rpm/test_daos_node.sh

```diff
@@ -95,25 +95,22 @@ if ! module load $OPENMPI; then
     module list
     exit 1
 fi
-coproc daos_server --debug start -t 1 --recreate-superblocks 2>&1
-trap 'set -x; kill -INT $COPROC_PID' EXIT
+coproc SERVER { daos_server --debug start -t 1 --recreate-superblocks; } 2>&1
+trap 'set -x; kill -INT $SERVER_PID' EXIT
 line=""
 while [[ "$line" != *started\ on\ rank\ 0* ]]; do
-  read -r -t 60 line <&"${COPROC[0]}"
+  read -r -t 60 line <&"${SERVER[0]}"
```

Comment:
Given the opportunity to look over this code more closely, this would probably be helpful:

```suggestion
if ! read -r -t 60 line <&"${SERVER[0]}"; then
    rc=${PIPESTATUS[0]}
    if [ "$rc" = "142" ]; then
        echo "Timed out waiting for output from the server"
    else
        echo "Error reading the output from the server: $rc"
    fi
    exit "$rc"
fi
```

Certainly not required for this PR but would be good to add at some time.  If I only had bandwidth to spin another plate.